frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-item-details.tsx

```diff
@@ -349,13 +372,30 @@ export const OperatorHubItemDetails: React.FC<OperatorHubItemDetailsProps> = ({
               />
             </PropertiesSidePanel>
             <div className="co-catalog-page__overlay-description">
+              {isAWSSTSCluster(cloudCredentials, infrastructure, authentication) && showWarn && (
+                <Alert
+                  isInline
+                  variant="warning"
+                  title={t('olm~Cluster in STS Mode')}
+                  actionClose={<AlertActionCloseButton onClose={() => setShowWarn(false)} />}
```

Comment:
```suggestion
                  actionClose={<AlertActionCloseButton onClose={() => setShowWarn(false)} />}
                  className="pf-u-mb-lg"
```

To improve spacing between `Alert` and start of contents.